FM636A/範例程式/CH03/upload/index.html

```diff
@@ -17,10 +17,10 @@
   <canvas id="myCanvas" width="1000" height="500"
     style="border:3px solid #d3d3d3c5;"></canvas>
   <script>
+    var bg_color = '#b2fffe';
     function canvas_arrow(context, fromx, fromy, tox, toy) {
       var headlen = 250; // length of head in pixels
       var angle = Math.atan2(toy - fromy, tox - fromx);
-      var bg_color = '#b2fffe';

```

Comment:
var bg_color was put in a wrong place, causing the font lighting function does not work.